### ISSUE TITLE
Directory naming scheme for IETF specs?

### ISSUE THREAD

**Issue by foolip** [w3c/web-platform-tests#10837]
We have at least 3 directories for IETF specs:
https://github.com/w3c/web-platform-tests/tree/master/cookies
https://github.com/w3c/web-platform-tests/tree/master/http
https://github.com/w3c/web-platform-tests/tree/master/x-frame-options

Perhaps more. @alvestrand is considering adding tests for https://tools.ietf.org/html/draft-ietf-rtcweb-jsep-24 but what should the directory be called?

**Comment by foolip** [comment]
Pinging owners of those existing directories: @inikulin @mikewest @annevk @mnot

**Comment by alvestrand** [comment]
Possibilities I've considered include webrtc/jsep/test (linking it to the w3c spec that uses it), rtcweb/jsep/test (making more room for other rtcweb WG-related specs) and jsep/test.

**Comment by mikewest** [comment]
*shrug* Call it whatever makes it most clear to developers what it's actually testing? It doesn't make much sense to me to lean too much on the place where the spec was written. We already have a `webrtc` directory. Throwing it under there seems fine? Having both `webrtc` and `rtcweb` would seem strange.

**Comment by foolip** [comment]
Sent https://github.com/w3c/web-platform-tests/pull/10838 (needs review) and https://github.com/w3c/web-platform-tests/pull/10839 to add a READMEs to at least point to the IETF specs under test, probably got it wrong :)

**Comment by alvestrand** [comment]
When actually writing a test to be put into the proposed directory, I thought that "webrtc/protocol" looked like a fine directory name - it's protocol specs rather than API specs, and its strictly related to WebRTC.
https://chromium-review.googlesource.com/c/chromium/src/+/1043886 - I guess the bot will come up with a github review CL soon.

**Comment by annevk** [comment]
FWIW, I think that's reasonable.

**Comment by jgraham** [comment]
Yeah, I don't have a strong opinion here. I don't know of any generic tooling that particularly relies on the naming (CSS specific tools aren't generic), so we should focus on clarity.

**Comment by annevk** [comment]
(And it does seem useful to put URLs to the specification in a README, ideally the same URLs used to reference the standard from other standard, so you can grep for it.)
